homeassistant/components/zha/core/channels/general.py

```diff
@@ -247,11 +248,11 @@ def __init__(
     ) -> None:
         """Initialize OnOffChannel."""
         super().__init__(cluster, ch_pool)
-        self._state = None
+        self._state: Optional[bool] = None
         self._off_listener = None
```

Comment:
I'm not positive, but I think this could be ` self._off_listener: Callable[..., None] = None`